src/oss/confluent.sh

```diff
@@ -95,6 +95,14 @@ declare -a commands=(
     "config"
 )
 
+declare -a dynamic_cli_commands=(
+    "sr-acl"
```

Comment:
The proposal now is have 'acl' as command and 'sr' as the subcommand so that we are future proof. We just don't want to have any implicit default since we will sooner or later have the Kafka ACL integrated into this.